File: backend/apps/users/urls.py
from django.urls import path

from users.views.change_password import ChangePasswordView
from users.views.confirmation_code import SendConfirmationCodeView, CheckConfirmationCodeView
from users.views.device_token import DeviceTokenView
from users.views.forgot_password import ForgotPasswordView
from users.views.sign_in import SignInView
from users.views.sign_out import SignOutView
from users.views.sign_up import SignUpView
from users.views.users import UserAllListView, UserDetailView

urlpatterns = [
    path('users', UserAllListView.as_view(), name='users'),
    path('user/<int:pk>', UserDetailView.as_view(), name='user'),
    path('send-confirmation-code', SendConfirmationCodeView.as_view(), name='send-confirmation-code'),
    path('check-confirmation-code', CheckConfirmationCodeView.as_view(), name='check-confirmation-code'),
    path('sign-up', SignUpView.as_view(), name='sign-up'),
    path('sign-in', SignInView.as_view(), name='sign-in'),
    path('sign-out', SignOutView.as_view(), name='sign-out'),
    path('forgot-password', ForgotPasswordView.as_view(), name='forgot-password'),
    path('change-password', ChangePasswordView.as_view(), name='change-password'),
    path('device-token', DeviceTokenView.as_view(), name='device-token'),
]

File: reactMobile/src/urls.js
// CORE
export const SEND_NOTIFICATION = '/core/send-notification'

// USER
export const SEND_CODE = '/users/send-confirmation-code'
export const CONFIRM_CODE = '/users/check-confirmation-code'
export const SET_PASSWORD = '/users/set-password'
export const SIGN_UP = '/users/sign-up'
export const SIGN_IN = '/users/sign-in'
export const SIGN_OUT = '/users/sign-out'
export const CHANGE_PASSWORD = '/users/change-password'

export const COUNTRY = '/users/country'
export const REGION = '/users/region'

export const PROFILES = '/users/profiles'
export const PROFILE = '/users/profile/{id}'
export const PROFILE_IMAGES = '/users/profile/images'
export const PROFILE_IMAGE = '/users/profile/image/{id}'
export const CHANGE_PROFILE_IMAGES = '/users/profile/change_images'
export const SINGLE_PROFILE_IMAGE = '/users/profile/image'

export const LIKES = '/users/likes'
export const LIKE = '/users/like/{id}'
export const DISLIKES = '/users/dislikes'
export const DISLIKE = '/users/dislike/{id}'
export const NUM_OF_LIKES = '/users/num-of-likes'

export const DEVICE_TOKEN = '/users/device-token'

// CHAT
export const ROOMS = '/chat/rooms'
export const ROOM = '/chat/room/{id}'
export const MESSAGES = '/chat/messages'
export const UNSEEN_MESSAGES = '/chat/unseen-messages'
